changelog/@unreleased/pr-4879.v2.yml

```diff
@@ -0,0 +1,40 @@
+type: improvement
+improvement:
+  description: |-
+    add refresh failure callbacks
```

Comment:
changelog description should be between a sentence and a paragraph, not the PR template.

Reply:
Done